spring-cloud-starter-stream-sink-field-value-counter/README.adoc

```diff
@@ -61,5 +75,16 @@ $$spring.redis.url$$:: $$Redis url, which will overrule host, port and password
 == Build
 
 ```
-$ mvn clean package
+$ ./mvnw clean install -PgenerateApps
+$ cd apps
+
+You can find the corresponding binder based projects here. You can then cd into one one of the folders and build it:
```

Comment:
I think we should consider to present this sentence out of the code block.
See my change for the `aggregator`: https://github.com/spring-cloud-stream-app-starters/aggregator/pull/9/files#diff-84c171210225cbda875c9ddc438ce2abR108